index.html

```diff
@@ -69,21 +70,21 @@ <h2 class="hidden">Ingredients</h2>
                 <input id="Option9" type="checkbox" autocomplete="off"><label class="chk" for="Option9">2 egg yolks</label>
                 <input id="Option10" type="checkbox" autocomplete="off"><label class="chk" for="Option10">1 lemon, juiced</label>
                 <input id="Option11" type="checkbox" autocomplete="off"><label class="chk" for="Option11">2 tbsp butter</label>
-                <input id="Option12" type="checkbox" autocomplete="off"><label class="chk" for="Option12">1 cup honey roasted pecan, roughly chopped</label>
+                <input id="Option12" type="checkbox" autocomplete="off"><label class="chk" for="Option12">1 cup honey roasted pecans, <span class="push">roughly chopped</span></label>
               </form>
-            </div>
+            </section>
+
+            <aside>
+              <img src="imgs/headshot.png" alt="vanessa headshot" class="left headshot">
+              <h3>Vanessa Stevenson</h3>
+              <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
+              <a href="#"><p class="share right">SHARE RECIPE</p></a>
```

Comment:
Change this to:
```
<p><a href="">Share Recipe</a></p>
```
or simply (with attendant styling)
```
<a href="" class="button">Share Recipe</a>
```
There's some strange things happening here if you open your inspector due to the wrapping.